src/LoginPage.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import logo from './logo.svg';
+import './LoginPage.css';
+
+function LoginPage() {
+	return (
+		<div className="Login-page-container">
+			<img src={logo} className="Passr-logo" alt="logo" />
+			<form>
+				<label className="Email-label">Email*</label>
+				<input type="text" id="Email" className="Email"/>
+				<label className="Password-label">Password*</label>
+  			<input type="text" className="Password"/>
+			</form>
+			<button type="button" className="Login-button">Login</button>
+      <a
+        className="Password-link"
+        href=""
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        Forgot your password?
+      </a>
+			<a
+        className="New-user-link"
+        href=""
```

Comment:
Your build will pass if you put in a dummy url, you can use ` href="https://reactjs.org"` for now if you want to.

Reply:
Thank you! I'll make sure to add those when I next push up!